dom.bs

```diff
@@ -10101,4 +10101,11 @@ Per <a rel="license" href="//creativecommons.org/publicdomain/zero/1.0/">CC0</a>
 the extent possible under law, the editors have waived all copyright and related or
 neighboring rights to this work.
```

Comment:
Maybe add a closing `</p>` here so the `<script>` doesn't end up inside the `<p>`? It doesn't really matter, but it seems better...